runtime/server/auth/claims.go

```diff
@@ -74,3 +85,30 @@ func (c anonClaims) Can(p Permission) bool {
 func (c anonClaims) CanInstance(instanceID string, p Permission) bool {
 	return false
 }
+
+func (c anonClaims) Attributes() map[string]any {
+	return nil
+}
+
+// devJWTClaims implements Claims and allows all actions.
+// It is used for servers with auth disabled.
```

Comment:
Suggestion: "It is used for mimicking user attributes on local when auth is disabled"

(because `openClaims` is used in regular cases when auth is disabled)